src/ORM/Table.php

```diff
@@ -788,16 +788,17 @@ public function behaviors(): BehaviorRegistry
      */
     public function getBehavior(string $name): Behavior
     {
-        /** @var \Cake\ORM\Behavior|null $behavior */
-        $behavior = $this->_behaviors->get($name);
-        if ($behavior === null) {
+        if (!$this->_behaviors->has($name)) {
```

Comment:
Instead of this `has()` check we could just call `get()` in a try / catch block and re-throw the exception with specific message if `has()` throws exception.

In majority of the cases the `has()` call will succeed and hence the extra method call will be avoided.

Reply:
Try catch is probably slower and i find it for code flow usually less nice than normal methods.